docker-storage-setup.sh

```diff
@@ -410,6 +412,19 @@ check_docker_storage_metadata() {
   exit 1
 }
 
+reset_docker_root_volume () {
+  local mp
+  if docker_root_volume_exists; then
+     if ! mp=$(docker_root_lv_mountpoint); then
+        return 1
```

Comment:
It will be a good idea to give an error message here. Say

`Failed to determine where /dev/$VG/$DOCKER_ROOT_LV is mounted`